tests/unit/branch_name_test.py

```diff
@@ -0,0 +1,63 @@
+import branch_name_verification_hook as branch_name_verification_hook
+from branch_name_verification_hook import *
+
+excluded_branches = {'test-1', 'sdfsdfs', '!@dvsfosk*&*&^fs4943'}
+
+def test_is_excluded_branch():
+    """
+    test excluded branches checking
+    """
+    assert not is_excluded_branch('test', excluded_branches)
+    assert is_excluded_branch('test-1', excluded_branches)
+    assert is_excluded_branch('!@dvsfosk*&*&^fs4943', excluded_branches)
+
+def test_get_issue_num():
+    """
+    test get issue number function
+    """
+    assert get_issue_num('upgrade-ubuntu-18-304') == '304'
```

Comment:
I suggest some negative cases:
- `test`
- `18-test`
- `-18-test-735-51-&`

The function should return `None` for these cases

Reply:
Added